bundles/org.openhab.binding.solax/README.md

```diff
@@ -0,0 +1,210 @@
+|--------------------------|----------------------------|-------------------------------------------------------------------------------------------|
+| inverterOutputPower      | system.electric-power      | The output power of the inverter [W]           |
+| inverterCurrent          | system.electric-current    | The output current of the inverter [A]  |
+| inverterVoltage          | system.electric-voltage    | The output voltage of the inverter [V]    |
+| inverterFrequency        | Number:ElectricPotential   | The frequency of the output voltage [Hz] |
+
+### Photovoltaic Panels Production Channels
+
+| Channel                  | Type                       | Description                            |
+|--------------------------|----------------------------|----------------------------------------|
+| pv1Voltage      | system.electric-voltage    | The voltage of PV1 string [V]                   |
+| pv2Voltage      | system.electric-voltage    | The voltage of PV2 string [V]                   |
+| pv1Current      | system.electric-current    | The current of PV1 string [A]                   |
+| pv2Current      | system.electric-current    | The current of PV2 string [A]                   |
+| pv1Power        | system.electric-power      | The output power PV1 string [W]                 |
+| pv2Power        | system.electric-power      | The output power PV2 string [W]                 |
+| pvTotalPower    | system.electric-power      | The total output power of both PV strings [W]   |
+| pvTotalCurrent  | system.electric-current    | The total current of both PV strings [A]        |
+
+### Battery channels
+
+| Channel                  | Type                       | Description                                                                                    |
+|--------------------------|----------------------------|------------------------------------------------------------------------------------------------|
+| batteryPower             | system.electric-power      | The power to / from battery (negative means power is pulled from battery and vice-versa) [W]   |
+| batteryCurrent           | system.electric-current    | The current to / from battery (negative means power is pulled from battery and vice-versa) [A] |
+| batteryVoltage           | system.electric-voltage    | The voltage of the battery [V]                                                                 |
+| batteryTemperature       | Number:Temperature         | The temperature of the battery [C/F]                                                           |
+| batteryStateOfCharge     | system.battery-level       | The state of charge of the battery [%]                                                         |
+
+### Grid related channels
+
+| Channel                  | Type                       | Description                                                                                    |
```

Comment:
Can the formatting of this table (in source form) be fixed?

Reply:
I hope I fixed them all...